config/bookie.truststore.jks.passwd

```diff
@@ -0,0 +1 @@
+1111_aaaa
```

Comment:
what is this? Is this needed for this PR?

Reply:
I think this deserves some explanation. Is this here for testing purposes? default trust store? or what? Can we add a comment to this to explain how this password is used?

Reply:
Added a comment.

Reply:
This is default and also reused for testing. Do you recommend making a copy of it for testing?

Reply:
I think it is odd to have a default password. I can understand having one for testing purposes, but not a default password for users. If you think that for educational purposes we should have the file there (so that a new users see that we require a password for the truststore, then I suggest we leave the file there but remove the password in it. Leave the comment and expand to say that it needs to be populated. We additionally need a copy for test purposes with a test password.

What do you think?

Reply:
These passwords are used for JKS keystore/truststore. As we provide a default truststore and keystore, it makes sense to provide a password file to access these as well.